docker-compose.yaml

```diff
@@ -1,9 +1,17 @@
 version: '2'
 services:
+  db:
+    image: postgres
```

Comment:
should not we have specific version of postgres here ? I guess it means :leatest

Reply:
done